### PR TITLE
GEOMESA-1841 Update HBase heatmap docs for geomesa:density and coproc…

### PR DESCRIPTION
**Pull request by aheyne**
…essor addition

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

**Comment by jnh5y** [comment]
@mdzimmerman any feedback?  Otherwise, it looks good to me!

File: docs/user/process.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.. _GeoMesa-Process:
```
Comment (elahrvivaz) [inline docs/user/process.rst]: [picky] let's stick with `_geomesa_process` for link consistency

File: docs/user/hbase/heatmaps.rst [hunk 2]
```diff
@@ -1,10 +1,34 @@
 HBase Heatmaps
 ==============
 
-HBase can generate client-side heatmaps in GeoServer by defining a style and using the built in ``gs:Heatmap`` process
-within GeoServer.
+GeoMesa on HBase can leverage server side processing to accelerate heatmap (density) queries. GeoMesa uses a custom
+coprocessor running on the HBase Region Servers so that density calculation are done at the data. The per-region results
+are then returned to GeoServer, combined there, and served. This functionality can be use accessed using the
```
Comment (mdzimmerman) [inline docs/user/hbase/heatmaps.rst]: "use accessed"?

File: docs/user/hbase/heatmaps.rst [hunk 3]
```diff
@@ -1,10 +1,34 @@
...
+-------------------
+
+In order to style the density result to produce a heatmap, GeoServer needs a Style Layer Descriptor (SLD).
 
 To start, add a new SLD style to GeoServer named "heatmap" and use this sld
 :download:`heatmap.sld </user/_static/sld/clientside-heatmap.sld>`
 
+Instruction on how to install SLDs can be found here `GeoServer Styles <http://docs.geoserver.org/latest/en/user/styling/webadmin/index.html>`_.
+
 Now you can change the style parameter to be styles=heatmap in the Layer Preview or PNG preview in GeoServer.
```
Comment (mdzimmerman) [inline docs/user/hbase/heatmaps.rst]: [picky] enclose "styles=heatmap" in backticks?

File: docs/user/hbase/heatmaps.rst (hunk 2 above)
Comment (aheyne) [inline docs/user/hbase/heatmaps.rst]: fixed

File: docs/user/hbase/heatmaps.rst (hunk 3 above)
Comment (aheyne) [inline docs/user/hbase/heatmaps.rst]: Fixed

**Pull request merged**